### DISCUSSION TITLE
Please revert change #36353 to time series in update 8.0.5. It ruins my use-case.

### DISCUSSION THREAD

**Discussion by westonforbes** [grafana/grafana#54039] [Dashboards, Panels & Visualizations]
Please revert change #36353 to time series in update 8.0.5.

 It removed the ability to view data in the tool tip that was not displayed on the time series itself. The reasoning given by @dprokop was

> It doesn't make much sense to have series shown in the tooltip when it's not visible in the visualization. This will skip series in the tooltip in such cases now.

However, its quite useful to have data shown in the tooltip that is not displayed. I am using the time series as a visual representation of records that have been logged to the server. The graph area itself shows the primary information of if the machine is running or not. I'd like the user to be able to hover over a record and see detailed information about that record, such as part count, job number, etc.

![image](https://user-images.githubusercontent.com/56303146/185948990-60b9f76a-46df-4896-b8b7-f8897e27d0af.png)

I submitted issue #53751 reporting this as a bug but it was closed by @mellieA as duplicate (referencing this update) and saying that to get the intended behavior I should set my series opacity to zero. However if the series is displayed (with opacity 0) this messes up all the axis auto ranging limits. If I want to be ranging around data that hovers between 0 and 10 then I have a hidden part count series of 20,000, that ruins the graph.

Please re-enable the ability to hide a series in the graph but not in the tool tip. It was a good feature and the proposed work-around does not work for all use cases.
Thank you.

**Comment by dprokop** [comment]
@westonforbes thanks for opening the discussion. I have not noticed this PR brought couple of comments asking to bring the feature back. Let me take a look at this soon and come up with sth that will satisfy both your and https://github.com/grafana/grafana/issues/36290 needs. 

And - sorry for the trouble!

**Comment by westonforbes** [comment]
@dprokop, thank you. I appreciate the quick response and for looking into this.
I think being able to view detailed data at the tooltip would be useful for anyone blending data sets and types. Especially because one field I want to display at the tooltip is "part number" which is a string, which obviously won't show in a graph properly.

Thanks again.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
